exporter/collector/logs.go

```diff
@@ -52,6 +52,9 @@ const (
 	LogNameAttributeKey        = "gcp.log_name"
 	SourceLocationAttributeKey = "gcp.source_location"
 	TraceSampledAttributeKey   = "gcp.trace_sampled"
+
+	GCPTypeKey                 = "@type"
```

Comment:
Nit: Should this be `GCPErrorReportingTypeKey` or `GCPErrorTypeKey`  and `GCPErrorTypeValue`  - basically consistent naming that indicates its a key-value pair to be used together ?

Reply:
this key isn't specific to Error Reporting, any protobuf marshalled to json will use it to indicate the object type. Error Reporting lets you parse it manually to bypass setting required fields in the proto, which is what we're doing here.

Reply:
I see, that makes sense.